container-chains/templates/simple/runtime/src/lib.rs

```diff
@@ -608,6 +626,24 @@ impl tp_impl_tanssi_pallets_config::Config for Runtime {
     type AuthoritiesNotingWeights = pallet_cc_authorities_noting::weights::SubstrateWeight<Runtime>;
 }
 
+parameter_types! {
+    // One storage item; key size 32 + 20; value is size 4+4+16+20. Total = 1 * (52 + 44)
+    pub const DepositBase: Balance = currency::deposit(1, 96);
+    // Additional storage item size of 20 bytes.
+    pub const DepositFactor: Balance = currency::deposit(0, 20);
```

Comment:
These factors are assuming a 20 byte account id. So the are correct for the frontier template, but incorrect for simple template and tanssi, which use a 32 byte account id. You can copy the values from asset-hub: 

https://github.com/paritytech/polkadot-sdk/blob/6c5a42a690f96d59dbdf94640188f5d5b5cc47e2/cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs#L415

Reply:
Same in runtime/dancebox/src/lib.rs and runtime/flashbox/src/lib.rs